src/web-worker/amp-worker.js

```diff
@@ -72,12 +72,41 @@ class AmpWorker {
     // Use RTV to make sure we fetch prod/canary/experiment correctly.
     const useLocal = getMode().localDev || getMode().test;
     const useRtvVersion = !useLocal;
-    const url = calculateEntryPointScriptUrl(
-      loc,
-      'ww',
-      useLocal,
-      useRtvVersion
+
+    let url = '';
+
+    const policy = {
+      createScriptURL: function (url) {
+        // Only allow the correct webworker url to pass through
+        const regexURL = new RegExp(
+          // eslint-disable-next-line local/no-forbidden-terms
+          '^https://([a-zA-Z0-9_-]+.)?cdn.ampproject.org(/.*)?$'
+        );
+        const testRegexURL = new RegExp('^([a-zA-Z0-9_-]+.)?localhost$');
```

Comment:
Should `+.` be `+\.`? Also appears on the regex above

Reply:
Ah you're right. It looks like this was also in the extension-script PR but that was already merged in. Is there anyway to quickly edit that or do we want to open another PR to fix it?

Reply:
Yeah it is a bit too permissible but the code itself should already be safe. This is just being added as a secondary measure in case there is something we aren't seeing. I'll open a PR soon with a fix.

Reply:
Opened at 39273